howto/android/integrate-hidl.md

```diff
@@ -0,0 +1,108 @@
+(howto-integrate-hidl)=
+# How to integrate Anbox HIDL interface with a custom VHAL implementation
+
+The Anbox HIDL interface is used in the default Anbox VHAL implementation. This document guides you through the steps for setting up and integrating the Anbox HIDL interface with your custom VHAL implementation.
+
+Due to the technical requirements of this procedure, this document assumes familiarity with AOSP development on your part. If you are looking for a simpler solution to have a custom VHAL implementation, such as loading an existing VHAL implementation and making it work with Android runtime, see {ref}`replace the Anbox VHAL <howto-replace-anbox-vhal>`.
```

Comment:
@ajanon Could you help me with verifying if this phrasing is technically accurate?

Reply:
> Due to the technical requirements of this procedure, this document assumes familiarity with AOSP development on your part.

:+1: 

> If you are looking for a simpler solution to have a custom VHAL implementation, such as loading an existing VHAL implementation and making it work with Android runtime, see {ref}`replace the Anbox VHAL <howto-replace-anbox-vhal>`.

Well, that's not exactly true. If users want to replace the default VHAL implementation with their own, they **must** replace the Anbox VHAL by following that link. (or use a custom system image)